cdm/src/test/java/ucar/nc2/ft/simpgeometry/TestCFSimpleGeometryCont.java

```diff
@@ -0,0 +1,319 @@
+package ucar.nc2.ft.simpgeometry;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import ucar.nc2.ft2.coverage.simpgeometry.*;
+
+import java.util.Random;
+import java.util.List;
+
+/**
+ * Very simple tests for Simple Geometry container objects.
+ * 
+ * @author wchen@usgs.gov
+ *
+ */
+public class TestCFSimpleGeometryCont {
+	

```

Comment:
Most of the logic in the implementation and the test approximates the logic of the native LinkedList class. Is there a way to simplify the implementation and the test by using the native LinkedList class instead?

Reply:
The motivation for using a Polygon class with next and previous to represent multipolygons was so that they could be treated alike and hence be placed in the same containers since they are very similar (a multipolygon is basically the same except with a "next" while a regular polygon has null "next").